helm/reference-wallet/templates/backend-deployment.yaml

```diff
@@ -229,8 +222,6 @@ spec:
           value: {{ .Values.vaspComplianceKey | toJson | quote }}
         - name: JSON_RPC_URL
           value: {{ .Values.sdk.jsonRpc }}
-        - name: OFFCHAIN_SERVICE_PORT
-          value: {{ .Values.service.targetOffchainPort | quote }}
         - name: VASP_BASE_URL
           value: http://{{ .Values.ingress.host }}/offchain
```

Comment:
Does this need to change too?

Reply:
good catch, probably need to be http://{{ .Values.ingress.host }}/api/offchain